dvc/command/cache.py

```diff
@@ -3,18 +3,14 @@
 import argparse
 
 from dvc.command.base import fix_subparsers, append_doc_link
-from dvc.command.remote import CmdRemoteAdd
 from dvc.command.config import CmdConfig
+from dvc.remote.config import cache_dir
 
 
 class CmdCacheDir(CmdConfig):
     def run(self):
-        self.args.name = "cache.dir"
-        self.args.value = CmdRemoteAdd.resolve_path(
-            self.args.value, self.configobj.filename
-        )
-
-        return super(CmdCacheDir, self).run()
+        cache_dir(self.config, self.args.value, level=self.args.level)
```

Comment:
name `cache_dir` is confusing. Should it be `set_cache_dir` at least? It's also confusing that remote handles cache dir - very counterintuitive, even though I understand where it comes from. In 90% or even 95% it will be called to set a dir for the workspace cache, which is not even local remote cache (or at least should not be?)

Reply:
It is actually about local remote cache, nothing else.